client/app/scripts/superdesk-authoring/metadata/views/metadata-dropdown.html

```diff
@@ -1,9 +1,9 @@
 <div class="dropdown" ng-class="{'dropright': icon, 'dropdown-noarrow': !icon}" dropdown sd-dropdown-position>
-    <button class="dropdown-toggle line-input" dropdown-toggle ng-if="field !== 'place' && !icon">
+    <button class="dropdown-toggle line-input" dropdown-toggle ng-if="(field !== 'place' && field !== 'genre') && !icon">
```

Comment:
I  think the condition should be `ng-if="(field !== 'place' || field !== 'genre') && !icon"` not `ng-if="(field !== 'place' && field !== 'genre') && !icon"`